ser574-green-team/taigit
# Us 204 project page

Project selection is now possible! On the Projects page, you can authenticate with Github, connect with Taiga, and then select projects to display using the dropdown buttons. The project selection is now hooked up to the components on the GitHub and Taiga page, so you should be able to change which projects you want to look at and the graphs will change to match.

To test:
1. Open the Project page and click Authenticate with Github and follow the instructions.
2. Back on the Project page, add your Taiga username and password, then press "Connect Taiga Account".
3. Ignore the Codacy input button. That's not working yet. Congrats, you've now connected your accounts to our app!
4. Use the two dropdown menus to select which Github and Taiga projects you want to look at. You can press the Analyze Project button, but it won't do anything. Your project is actually being set when you make the selection in the dropdown.
5. Switch to the other tabs. GitHub and Taiga pages should show the right data if those components were already connected to the API. I refactored most components, but there are still some with dummy data.
6. The titles of each page should now show the names of the project.
7. Try changing the projects in the selectors and see if everything shows up correctly.
8. Profit.

Sometimes things don't show up right away, so try refreshing the page if something isn't appearing. If you have components in other branches that are still using hardcoded data, let me know and I can help you pull in the right data for your function calls.
